tests/unit-tests/tests-user-functions.php

```diff
@@ -230,5 +230,41 @@ function test_give_validate_user_password() {
 
 		$this->assertTrue( $output );
 	}
+
+	/**
+	 * Test give_donor_email_exists function
+	 *
+	 * @since 1.8.9
+	 *
+	 * @cover give_donor_email_exists
+	 */
+	function test_give_donor_email_exists() {
+		/*
+		 * Check 1
+		 *
+		 * Empty email doesn't exists
+		 */
+		$output = give_donor_email_exists( '' );
+		$this->assertFalse( $output );
+
+		/*
+		 * Check 2
+		 *
+		 * Bad Email doesn't exists
+		 */
+		$output = give_donor_email_exists( 'xyz' );
+
+		$this->assertFalse( $output );
+
+		/*
+		 * Check 3
+		 *
+		 * Email already exists for either donor or user
+		 */
+		$output = give_donor_email_exists( 'admin@example.org' );
+
+		$this->assertTrue( $output );
+
```

Comment:
@mehul0810 can we add a todo here?
basically currently we are not validation this function with additional email, we will come back and update this test soon. Our test needs to update